lightningd/pay.c

```diff
@@ -1460,6 +1460,11 @@ static struct command_result *json_createonion(struct command *cmd,
 	for (size_t i=0; i<tal_count(hops); i++)
 		sphinx_add_hop(sp, &hops[i].pubkey, hops[i].raw_payload);
 
+	if (sphinx_path_payloads_size(sp) > ROUTING_INFO_SIZE)
+		return command_fail(
+		    cmd, LIGHTNINGD,
```

Comment:
Deserves its own unique error code in `common/jsonrpc_errors.h`? (also `doc/lightning-createonion.7.md`)  Or alternately `JSONRPC2_INVALID_PARAMS`.